pyanaconda/modules/localization/localed.py

```diff
@@ -223,7 +242,86 @@ def convert_layouts(self, layouts_variants):
         orig_layouts_variants = self.layouts_variants
         orig_keymap = self.keymap
         ret = self.set_and_convert_layouts(layouts_variants)
-        self.set_layouts(orig_layouts_variants)
+        self._set_layouts(orig_layouts_variants)
         self.set_keymap(orig_keymap)
 
         return ret
+
+    # TODO: rename to select_layout
```

Comment:
Still something to be handled or is it for a followup PR ? :)

Reply:
yes, I think it might be more appropriate but truth is not necessary.